dev/com.ibm.ws.jca.cm/resources/OSGI-INF/l10n/metatype.properties

```
@@ -36,6 +36,9 @@ connTimeout.desc=Amount of time after which a connection request times out. A va
 maxIdleTime=Maximum idle time
 maxIdleTime.desc=Amount of time a connection can be unused or idle until it can be discarded during pool maintenance, if doing so does not reduce the pool below the minimum size. A value of -1 disables this timeout.
 
+maxInUseTime=Remove long running in use connections
+maxInUseTime.desc=This property is a temporary workaround for application code that does not close connections. It configures an amount of time after which pool maintenance can invoke Connection.abort on a connection that is being used by the application. Stopping connections that are in use can result in unexpected and unpredictable threading errors. The proper solution is to update the application to always close connections and not use this property. If you must configure it as a workaround, choose a value of 20 minutes or more depending on the normal application transaction times. If transaction times are high, 10 minutes or more, then double your longest running transaction time for this value. A value of -1 disables this timeout.
```

```suggestion
maxInUseTime.desc=This property is a temporary workaround for application code that does not close connections. It configures an amount of time after which pool maintenance can invoke the Connection.abort method on a connection that the application is using. Stopping connections that are in use might result in unexpected and unpredictable threading errors. The proper solution is to update the application to always close connections and not use this property. If you must configure this property as a workaround, choose a value of 20 minutes or more, depending on the normal application transaction times. If transaction times are high, 10 minutes or more, then double your longest running transaction time for this value. A value of -1 disables this timeout.
```

not sure if Connection.abort is a method or property, adjust as needed. Providing a noun identifier for each code entity is helpful for translations.

---

yep, it is a method. Thanks for the suggestions.